src/interface/blas2_interface.hpp

```diff
@@ -350,15 +352,16 @@ typename sb_handle_t::event_t _trsv_impl(sb_handle_t& sb_handle, index_t _N,
       blas::make_sycl_iterator_buffer<int32_t>(sync_vec, sync_vec.size());
   auto sync = make_vector_view(sync_buffer, one::value(), sync_vec.size());
 
-  auto trsv =
-      make_trsv<subgroup_size, subgroups, is_upper, is_transposed, is_unit>(
-          vx, mA, vx, sync);
+  index_t unused;
```

Comment:
Why do we introduce the unused variable. What was wrong with the previous definition. It is better to use the previous definition here as it was more readable and obvious that adding unused variable and making them to use the same function

Reply:
Done in https://github.com/codeplaysoftware/sycl-blas/pull/434/commits/441067da63570046d6ce44e9f55a971ee9246927.